src/libraries/Common/tests/System/Net/Http/HttpClientHandlerTest.cs

```diff
@@ -21,11 +21,12 @@ namespace System.Net.Http.Functional.Tests
 #if WINHTTPHANDLER_TEST
     using HttpClientHandler = System.Net.Http.WinHttpClientHandler;
 #endif
-
     // Note:  Disposing the HttpClient object automatically disposes the handler within. So, it is not necessary
     // to separately Dispose (or have a 'using' statement) for the handler.
     public abstract class HttpClientHandlerTest : HttpClientHandlerTestBase
     {
+        public static bool IsNotWinHttpHandler = !IsWinHttpHandler;
```

Comment:
What does WinHttpHandler do for these inputs?  Is it doing it "wrong", and if so, is that an issue with WinHttpHandler or with winhttp?

Reply:
It hangs for the newly added LLA. I did packet capture and it is failing to even connect e.g. there is no SYN going out. So it seems like the trick with tho proxy does not work in this case for some reason. 
I could possibly add skip just for that one or split it to multiple cases but I was not sure how much we should twist because of winhttp.